pkg/console/controllers/util/util.go

```diff
@@ -57,3 +59,28 @@ func LabelFilter(labels map[string]string) factory.EventFilterFunc {
 		return true
 	}
 }
+
+type consoleOperatorController interface {
+	HandleManaged(context.Context) error
+	HandleUnmanaged(context.Context) error
+	HandleRemoved(context.Context) error
+	GetOperatorSpec() (*operatorv1.OperatorSpec, error)
```

Comment:
in which case is this different from what you'd get from `GetOperatorState()` from `operatorClient v1helpers.OperatorClient`?

Reply:
It's not really different, we could just pass the operatorClient as a third arg to HandleMangementState and remove this func from the interface.